containers/kubernetes-helm-k3d/.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,31 @@
+{
+	"name": "Kubernetes - k3d-in-Docker",
+	"dockerFile": "Dockerfile",
+	"runArgs": ["--privileged"],
+	"mounts": [
+		"source=k3d-var-lib-docker,target=/var/lib/docker,type=volume",
+		"source=k3d-config,target=/home/vscode/.k3d,type=volume",
```

Comment:
Same comment here with regards `/home/vscode`

Reply:
maybe we can use something like "~" or "$(USER)"?

Reply:
We can use `${containerEnv:HOME}` to represent HOME directory inside the container.
Reference: https://code.visualstudio.com/docs/remote/devcontainerjson-reference#_variables-in-devcontainerjson

Reply:
`containerEnv:HOME`/.k3d doesn't work.
But `/home/${remoteUser}/.k3d` works fine.